hooks/pre_gen_project.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+import sys
+
+
+def validate():
```

Comment:
Just a thought, you might need to sanitize `svc_name` (make sure there aren't spaces or dashes or other things not allowed by Python or URLs).

For example, I tried to make a service called `technote-init` and of course when I built it I got

```
error in sqre-uservice-technote-init setup command: ("EntryPoint must be in 'name=module:attrs [extras]' format", 'sqre-uservice-technote-init = uservice_technote-init:standalone')
```